tests/test_ablation.py

```diff
@@ -10,51 +10,42 @@
 
 import csv
 import json
-import os
```

Comment:
Why remove import?

Reply:
It was only being used for `os.unlink`, which isn't being used anymore. Now `pathlib.Path.unlink` is being used (but it's actually used in a helper method in `tests.utils` so that the input argument can be coerced to a `Path` (i.e. so that we don't have to care about whether the path is a `str` or a `Path`).